ksp_plugin_adapter/class_marshalers.cs

```diff
@@ -14,18 +13,18 @@ internal class InBodyParametersMarshaler : ICustomMarshaler {
 
     [StructLayout(LayoutKind.Sequential)]
     internal class BodyParametersRepresentation {
-      public string name;
-      public string gravitational_parameter;
-      public string reference_instant;
-      public string min_radius;
-      public string mean_radius;
-      public string max_radius;
-      public string axis_right_ascension;
-      public string axis_declination;
-      public string reference_angle;
-      public string angular_frequency;
-      public string reference_radius;
-      public string j2;
+      public string? name;
+      public string? gravitational_parameter;
```

Comment:
`name` and `gravitational_parameter` are `required` in the proto, why are they optional here?

Reply:
Because of the default construction at line 37, which is hard to avoid.